src/tests/ftest/datamover/container_copy.yaml

```
@@ -0,0 +1,66 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+    - server-C
+    - server-D
+    - server-E
+  test_clients:
+    - client-F
+    - client-G
+    - client-H
+timeout: 3600
+server_config:
+  name: daos_server
+  servers:
+    log_mask: ERR
+    bdev_class: nvme
+    bdev_list: ["0000:81:00.0"]
+    scm_class: dcpm
+    scm_list: ["/dev/pmem0"]
+pool:
+  mode: 146
+  name: daos_server
+  scm_size: 42000000000       # 42G
+  nvme_size: 700000000000     # 700G
+  svcn: 1
+  control_method: dmg
+container:
+  type: POSIX
+  control_method: daos
+ior:
+  client_processes:
+    np: 30
+  api: DFS
+  flags: "-w -k"
+  dfs_destroy: False
+  test_file: /testFile
+  repetitions: 1
+  signature: 5
+  transfersize_blocksize:
+    1M:
+      transfer_size: '1M'
+      block_size: '5G'         # creating aggregate file of size 150G
```

```suggestion
      block_size: '5G'         # creating aggregate file of size 5G
```

Minor, if you have to push again, the comment doesn't match.
(Unless the size is actually going to be `150G`